BraveShared/BraveStrings.swift

```diff
@@ -350,7 +350,8 @@ public extension Strings {
     public static let Terms_of_Use = NSLocalizedString("Terms of Use", comment: "Show Brave Browser TOS page from the Privacy section in the settings.")
     public static let Bookmarks_and_History_Panel = NSLocalizedString("Bookmarks and History Panel", comment: "Button to show the bookmarks and history panel")
     public static let Private_Tab_Title = NSLocalizedString("This is a Private Tab", comment: "Private tab title")
-    public static let Private_Tab_Body = NSLocalizedString("Even though sites you visit in private tabs are not saved locally, they do not make you anonymous or invisible to your ISP, your employer, or to the sites you are visiting.", comment: "Private tab details")
+    public static let Private_Tab_Body = NSLocalizedString("Private Tabs aren't saved in Brave, but they don't make you anonymous online. Sites you visit in a private tab won't show up in your history and their cookies always vanish when you close them — there won't be any trace of them left in Brave. Your mobile carrier (or the owner of the WiFi network or VPN you're connected to) can see which sites you visit and and those sites will learn your public IP address, even in Private Tabs.", comment: "Private tab details")
+    public static let Private_Tab_Details = NSLocalizedString("Using Private Tabs only changes what Brave does on your device, it doesn't change anyone else's behavior.\n\nSites always learn your IP address when you visit them. From this, they can often guess roughly where you are — typically your city. Sometimes that location guess can be much more specific. Sites also know everything you specifically tell them, such as search terms. If you log into a site, they'll know you're the owner of that account. You'll still be logged out when you close the Private Tabs because Brave will throw away the cookie which keeps you logged in.\n\nWhoever connects you to the Internet (your ISP) can see all of your network activity. Often, this is your mobile carrier. If you're connected to a WiFi network, this is the owner of that network, and if you're using a VPN, then it's whoever runs that VPN. Your ISP can see which sites you visit as you visit them. If those sites use HTTPS, they can't make much more than an educated guess about what you do on those sites. But if a site only uses HTTP then your ISP can see everything: your search terms, which pages you read, and which links you follow.\n\nIf an employer manages your device, they might also keep track of what you do with it. Using Private Tabs probably won't stop them from knowing which sites you've visited. Someone else with access to your device could also have installed software which monitors your activity, and Private Tabs won't protect you from this either.", comment: "Private tab detail text")
```

Comment:
Rather than using `/n`, probably better to use a multi-line string literal `"""`

Reply:
Just following suite based on other strings. Can we use multiline strings for NSLocalizedString keys?

Reply:
I assumed 🤷‍♂️ 
I guess we'll find out 😁